### PR TITLE
SD-94: Removed spaces before council names, and amended and deleted Coucils

### PR DESCRIPTION
**Pull request by SarahJaneLuff**
Jira Ticket: https://collaboration.homeoffice.gov.uk/jira/browse/SD-94

The following changes have been made:

1. Spaces removed before value and label in the following: 
- {value:"Christchurch and Poole Council",label:"Christchurch and Poole Council"}
- {value:"Mourne and Down District Council",label:"Mourne and Down District Council"}
- {value:"Banbridge and Craigavon Borough Council",label:"Banbridge and Craigavon Borough Council"}

2. Modified ‘Christchurch and Poole Council’ to ‘Bournemouth, Christchurch and Poole Council’.
3. Deleted ‘Bournemouth Borough Council’, ‘Christchurch Borough Council’ and ‘Borough of Poole’.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -394,7 +391,7 @@ const ukLocalAuthorities = [
 {value:"Limavady Borough Council",label:"Limavady Borough Council"},
 {value:"Derry City Council",label:"Derry City Council"},
 {value:"Newry",label:"Newry"},
-{value:" Mourne and Down District Council",label:" Mourne and Down District Council"},
+{value:"Mourne and Down District Council",label:"Mourne and Down District Council"},
```
Comment (zuzak) [inline index.js]: Should this be `Newry, Mourne and Down District Council`?

File: index.js [hunk 2]
```diff
@@ -405,7 +402,7 @@ const ukLocalAuthorities = [
 {value:"Antrim & Newtownabbey Borough Council",label:"Antrim & Newtownabbey Borough Council"},
 {value:"Ards and North Down Borough Council",label:"Ards and North Down Borough Council"},
 {value:"Armagh City",label:"Armagh City"},
-{value:" Banbridge and Craigavon Borough Council",label:" Banbridge and Craigavon Borough Council"},
+{value:"Banbridge and Craigavon Borough Council",label:"Banbridge and Craigavon Borough Council"},
```
Comment (zuzak) [inline index.js]: Should this be `Armagh City, Banbridge and Craigavon District Council`?

File: index.js (hunk 1 above)
Comment (SarahJaneLuff) [reply index.js]: Removed Newry, and ammended to Newry, Mourne and Down District Council.

File: index.js (hunk 2 above)
Comment (SarahJaneLuff) [reply index.js]: Removed Armagh City
Amended to Armagh City, Banbridge and Craigavon.

**Review by zuzak — COMMENTED** [review]
I don't have access to that Jira ticket to check -- but it looks fine to me :)

**Comment by agraham19** [comment]
Looks good to me

**Pull request merged**